example/angular_components_example/pubspec.lock

```diff
@@ -7,42 +7,44 @@ packages:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.32.2"
+    version: "0.32.4"
   angular:
     dependency: "direct main"
     description:
       name: angular
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "5.0.0-beta"
+    version: "5.0.0-beta+2"
   angular_ast:
     dependency: transitive
     description:
       name: angular_ast
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.5.3+3"
+    version: "0.5.4"
   angular_compiler:
     dependency: transitive
     description:
       name: angular_compiler
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.4.0-alpha+16"
+    version: "0.4.0-alpha+18"
   angular_components:
     dependency: "direct main"
     description:
-      name: angular_components
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "0.9.0-beta"
+      path: "."
+      ref: HEAD
+      resolved-ref: "3ef0f20f7afb1ed186c1b729ca37352914ae3fd3"
+      url: "https://github.com/dart-lang/angular_components.git"
+    source: git
```

Comment:
You probably want to get this from pub, not git.

Reply:
Absolutely, thanks!